packages/extension-points/README.md

```diff
@@ -0,0 +1,36 @@
+# @drupal/extension-points-bundler
+
+Tooling to bundle and transpile your JavaScript to be used with the AdminUI.
+
+## Installation
+
+Add this module to your project. All the bundling and transpile dependencies and settings are included.
+
+```
+yarn add @drupal/extension-points-bundler
+
+# Build
+./node_modules/.bin/js-drupal-build
+
+# Build for production
+./node_modules/.bin/js-drupal-build --mode=production
+```
+
+## Documentation
+
+This tooling looks for your `*.admin_ui.components.yml` file and parses it's contents to create entry points for Webpack.
+
+This tooling assumes that your source JavaScript exists at `/js/src` and it's output will be `/js/src`.
```

Comment:
is `js/src` the correct output folder?

Reply:
yes it is: ```
  .then(({ components }) =>
    Object.keys(components.widgets).reduce(
      (acc, widgetName) => ({
        ...acc,
        ...{
          [`${widgetName}.widget`]: `./js/src/${basename(
            components.widgets[widgetName].component,
          )}`,
        },
      }),
```

Reply:
shouldn't we be building to something outside of `src`?

Reply:
Oh I totally misread you